root/layout/components/Footer.js

```diff
@@ -9,87 +9,568 @@
+                </a>
+              </li>
+              <li>
+                <img
+                  alt="Arrow"
+                  height="24"
+                  src="../../static/images/icons/angle_double_right_icon.svg"
+                  width="18"
+                />
+                <a
+                  className="fw-bold fs-4"
+                  href="https://musicbrainz.org/doc/Style"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('Style Guidelines')}
+                </a>
+              </li>
+              <li>
+                <img
+                  alt="Arrow"
+                  height="24"
+                  src="../../static/images/icons/angle_double_right_icon.svg"
+                  width="18"
+                />
+                <a
+                  className="fw-bold fs-4"
+                  href="https://musicbrainz.org/doc/How_To"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('How Tos')}
+                </a>
+              </li>
+              <li>
+                <img
+                  alt="Arrow"
+                  height="24"
+                  src="../../static/images/icons/angle_double_right_icon.svg"
+                  width="18"
+                />
+                <a
+                  className="fw-bold fs-4"
+                  href="https://musicbrainz.org/doc/Frequently_Asked_Questions"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('FAQs')}
+                </a>
+              </li>
+              <li>
+                <img
+                  alt="Arrow"
+                  height="24"
+                  src="../../static/images/icons/angle_double_right_icon.svg"
+                  width="18"
+                />
+                <a
+                  className="fw-bold fs-4"
+                  href="https://musicbrainz.org/doc/MusicBrainz_Documentation"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('Doc Index')}
+                </a>
+              </li>
+              <li>
+                <img
+                  alt="Arrow"
+                  height="24"
+                  src="../../static/images/icons/angle_double_right_icon.svg"
+                  width="18"
+                />
+                <a
+                  className="fw-bold fs-4"
+                  href="https://musicbrainz.org/doc/Development"
+                  rel="noopener noreferrer"
+                  target="_blank"
+                >
+                  {l('Development')}
+                </a>
+              </li>
+
+            </ul>
+          </div>
+        </div>
+        <div className="row mt-4">
+          <div className="col-md-3 border-top pt-4 d-none d-md-block fs-4">
+            <p>
+              {l('OSS Geek? ')}
```

Comment:
I noticed there are several strings with errant trailing whitespace (please search for them in your editor).
```suggestion
              {l('OSS Geek?')}
```

Reply:
Right!